java/src/jmri/NameIncrementingManager.java

```diff
@@ -0,0 +1,42 @@
+package jmri;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Interface that indicates that a Manager class has a
+ * {@link #getNextValidSystemName(NamedBean currentBean)} method that
+ * can be called without arguments.
```

Comment:
I find this sentence contradictory. The comment says that `getNextValidSystemName()` takes the currentBean as a parameter, but that the method can be called without arguments???

Or is the intent that `getNextValidSystemName()` can be called with `null` as parameter? In that case, it would be good to mark currentBean with `@CheckForNull`.

Another option which I use often in LogixNG is to add two methods, `getNextValidSystemName()` and `getNextValidSystemName(NamedBean)` there the first calls the second with `null` as parameter.

Reply:
Thanks.  That was left over inadvertently from the prior method's Javadoc, now fixed.